### PR TITLE
ISRE-508 Updating documentation to include json to yaml instructions.

### PR DESCRIPTION
**Pull request by coryleeio**

### CODE REVIEW

**Comment by coryleeio** [comment]
relevant link as well:
https://openedx.atlassian.net/wiki/spaces/AC/pages/1822916664/How+to+convert+your+lms+and+studio+json+configuration+files+to+yaml

File: en_us/open_edx_release_notes/source/juniper_developer.rst [hunk 1]
```diff
@@ -258,6 +258,22 @@ The aggregated data for many files is output as a csv.
 
 .. _pytest-repo-health: https://github.com/edx/pytest-repo-health
 
+
+==========================
+LMS / Studio Configuration
+==========================
+
+JSON to YAML
```
Comment (coryleeio) [inline en_us/open_edx_release_notes/source/juniper_developer.rst:266]: Let me know if this needs to be moved

File: en_us/open_edx_release_notes/source/juniper_developer.rst [hunk 2]
```diff
@@ -258,6 +258,22 @@ The aggregated data for many files is output as a csv.
...
+--------------------
+
+Most edx applications read a single YAML file.  However the lms and studio historically
+read multiple JSON ones. We are making the lms and studio behave the same as the other applications
+by having them read a single YAML file instead of multiple JSON ones. Below is a link to assist if you have difficulty
+with that transition. 
+
+.. __: https://openedx.atlassian.net/wiki/spaces/AC/pages/1822916664/How+to+convert+your+lms+and+studio+json+configuration+files+to+yaml
```
Comment (coryleeio) [inline en_us/open_edx_release_notes/source/juniper_developer.rst:275]: This link contains the technical bits regarding the transition

File: en_us/open_edx_release_notes/source/juniper_developer.rst (hunk 1 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/juniper_developer.rst:266]: This is a fine place. I tweaked it in #1886

Comment (coryleeio) [reply en_us/open_edx_release_notes/source/juniper_developer.rst:266]: Cool - merged

**Pull request merged**